.travis.yml

```diff
@@ -1,16 +1,8 @@
 language: csharp
 mono: latest
-dotnet: 2.0.0
+dotnet: 2.1.4
 dist: trusty
-
-addons:
-  apt:
-    sources:
-    - sourceline: 'deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-trusty-prod trusty main'
-      key_url: 'https://packages.microsoft.com/keys/microsoft.asc'
-    packages:
-    - dotnet-hostfxr-1.0.1
```

Comment:
@bgrainger do you remember why these packages were/are required? The template builds without them on the current SDK.

Reply:
I don't. If Travis CI archives really old builds then we could probably compare before and after; it was almost certainly to work around a Travis build failure.

Reply:
Looks like I added that: https://github.com/Faithlife/FaithlifeUtility/pull/3/files